admin-console/src/main/webapp/src/components/EventTypes/EventTypeTable.tsx

```diff
@@ -24,6 +24,7 @@ interface EventTypeTableBaseProps {
     onCreateEventType: () => void;
     onEditEventType: (eventType: EventType) => void;
     onDeleteEventTypeModal: (eventType: EventType) => void;
+    onCreateEditInstantTemplate: (i: InstantTemplate) => void
```

Comment:
The second parameter should be optional 
```suggestion
    onCreateEditInstantTemplate: (i?: InstantTemplate) => void
```